composite/bitcoin-json-rpc/src/adapter.ts

```diff
@@ -0,0 +1,44 @@
+import JSONRPC from '@chainlink/json-rpc-adapter'
+import { Execute, AdapterRequest } from '@chainlink/types'
+import { Validator } from '@chainlink/external-adapter'
+import { Requester } from '@chainlink/external-adapter'
+
+const inputParams = {
+  url: false,
+  method: false,
+  params: false,
```

Comment:
`url`, `method` and `params` will be handled by JSONRPC adapter. You can just forward the input to the json adapter an it will handle it

Reply:
I just left them here, so this composite adapter can inherit the actual functionality of the json-rpc one.